include/dlaf/lapack_tile.h

```diff
@@ -27,6 +27,15 @@ namespace tile {
 
 // See LAPACK documentation for more details.
 
+/// Reduce a Hermitian definite generalized eigenproblem to standard form
+/// (solving inv(U**H)*A*inv(U) or inv(L)*A*inv(L**H))
+///
+/// @pre a must be a square Hermitian matrix
```

Comment:
hermitian matrix is enough, it is square by definition.

Reply:
@rasolca I was talking about this in my main comment.

> Tile wrappers for blas/lapack referring to Hermitian matrices, AFAICT, are valid also for Symmetric real matrices, but in the documentation we don't say this or at least it is implicit that Hermitian is "extended" also for real values. There may be discussion about how to address this (e.g. having just the hermitian call addressing both complex and real, having two separate calls, ...), but I'm quite sure it is something we don't want to address in this PR. Do you @rasolca @teonnik think it is worth opening an issue about this?

Reply:
@albestro, a real symmetric matrix is an Hermitian matrix, (and complex transposition is equivalent to transposition for real matrices) so I don't see any reason why we should separate the calls.

Reply:
I agree with you, but I'm not getting what has to be done in the documentation.

Do we want to explicit that it works for real symmetric like we did in other places (because also LAPACK make this distinction) or for us it is enough saying Hermitian to address both complex and real (symmetric) case?

It's not a blocking thing, it's just an genuine interest about what do in the documentation to make it uniform.